package_manifest.yaml

```diff
@@ -7,20 +7,22 @@ packages:
     katello_packages: {}
     katello_client_packages: {}
 
+# Packages in the foreman directory
```

Comment:
I feel like we've failed at naming if these comments are needed.

Reply:
They were a trick to let the YAML reformatting maintain the newline. In the final PR I'll drop this commit and discuss it in a separate commit. It did allow me to script the adding of packages to the manifest though so that was a lot less work.